Wox.Core/Configuration/IPortable.cs

```diff
@@ -0,0 +1,16 @@
+﻿
+namespace Wox.Core.Configuration
+{
+    public interface IPortable
+    {
+        void EnablePortableMode();
+        void DisablePortableMode();
+        void RemoveShortcuts();
```

Comment:
Many of the methods here are not used, why do we need them in the interface?
it looks like we use only 3 methods here

Reply:
they are used:
![image](https://user-images.githubusercontent.com/26427004/77245835-c4fb5300-6c75-11ea-9be4-7b7d07c73532.png)